### PR TITLE
[NBS1.0] Remove `UrlReader` related deprecations

### PR DESCRIPTION
**Pull request by camilaibs**
## Hey, I just made a Pull Request!

Ref: https://github.com/backstage/backstage/issues/25999

Remove deprecated `Url Reader` exports.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.examples.test.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ describe('fetch:template examples', () => {
   beforeEach(() => {
     mockDir.clear();
     action = createFetchTemplateAction({
-      reader: Symbol('UrlReader') as unknown as UrlReader,
+      reader: Symbol('UrlReader') as unknown as UrlReaderService,
```
Comment (camilaibs) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.examples.test.ts:74]: Should we update this symbol as well?

Comment (Rugvip) [reply plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.examples.test.ts:74]: Eh, bit of a weird pattern to start tbh. Can probably just leave as is I don't think it has any effect

**Review by Rugvip — APPROVED** [review]
Nice! 🎉 

image

👨‍🍳 👌

File: plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.test.ts [hunk 2]
```diff
@@ -78,7 +80,7 @@ describe('fetch:template', () => {
       workspace: {},
     });
     action = createFetchTemplateAction({
-      reader: Symbol('UrlReader') as unknown as UrlReader,
+      reader: Symbol('UrlReaderService') as unknown as UrlReaderService,
```
Comment (camilaibs) [inline plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.test.ts:83]:
```suggestion
      reader: Symbol('UrlReader') as unknown as UrlReaderService,
```

**Pull request merged**